Makefile

```
@@ -0,0 +1,16 @@
+BINARY_NAME=pdc-agent
+
```

I think you may want to define these targets as .PHONY as well - otherwise Make will look for files named <recipe_name> when executing.
```suggestion
.PHONY: all build test lint clean
```

---

Nice one, thank you!